web/src/components/molecules/Schema/ModelFieldList.tsx

```diff
@@ -121,4 +125,10 @@ const ItemKey = styled.span`
   font-weight: 400;
 `;
 
+const ItemUnique = styled.span`
```

Comment:
Is there a reason you are using spans for all the text and not, for example, a p tag?

Reply:
It is just that span usually isn't used as the main tag for text. Usually it is used for styling sections of text, which is why I am not sure its a good idea to use it as a text element.
Of course using something like p you have to remove the margin, but since it is more meaningful syntactically I think it is better to use p or another text element and then change the size if necessary.
To be honest, unless it is a header or subheader (where h1 or h2 are obvious), I typically just use p and remove the margin. Not great, but still relatively syntactically meaningful

Reply:
Now the name is a `p` tag, and the others (unique indicator, etc..) are children of `p` tag so I kept them as `span`